### PR TITLE
added some of the new options to the example

### PR DESCRIPTION
**Pull request by CCOSTAN**
added in so that it was clearer that true/false were the values to use.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @CCOSTAN,

It seems you haven't yet signed a CLA. Please do so [here](https://cla.home-assistant.io/sign/CCOSTAN?pr=1905).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_components/light.hue.markdown [hunk 1]
```diff
@@ -27,6 +27,9 @@ If you want to enable the component without relying on the [discovery component]
 light:
   platform: hue
   host: DEVICE_IP_ADDRESS
+  allow_unreachable: true
+  allow_in_emulated_hue: false
+  allow_hue_groups: false
```
Comment (fabaff) [inline source/_components/light.hue.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Comment by robbiet480** [comment]
Hey, sorry the CLA link was wrong, it's now been fixed. Please sign the CLA [here](https://home-assistant.io/developers/cla_sign_start/).

**Comment by CCOSTAN** [comment]
Took out the examples.. added in (true/false) to options to show appropriate optional values.

**Pull request merged**